assets/src/apps/scheduler/DragBar.tsx

```diff
@@ -111,37 +113,38 @@ export const DragBar: React.FC<DragBarProps> = ({
       ? barGeometry(dayGeometry, workingStart, workingEnd)
       : barGeometry(dayGeometry, startDate, endDate);
 
-  const barStyles = {
-    left: geometry.left,
-    width: geometry.width,
-  };
-
-  const color = manual
-    ? 'bg-delivery-primary'
-    : 'bg-delivery-primary-300 dark:bg-delivery-primary-600';
-
   return (
     <>
       {isContainer ? (
         <div
           onMouseDown={startDrag}
-          className=" absolute border-t-4 border-black h-3 top-3 cursor-move flex flex-row justify-between dark:border-gray-400"
-          style={barStyles}
+          className={`absolute border-t-4 h-3 top-3 cursor-grab flex flex-row justify-between`}
+          style={{
+            left: geometry.left,
+            width: geometry.width,
+            borderTopColor: color,
+          }}
         >
           <div
             onMouseDown={startResize('left')}
-            className="w-1 inline-block h-full bg-black cursor-col-resize dark:bg-gray-400"
+            className={`w-1 inline-block h-full cursor-col-resize`}
```

Comment:
It seems like this className is not using any dynamic string interpolation, maybe it would make more sense to simply use className="..." syntax here (and instances below) instead?

Reply:
Done